src/Compilers/CSharp/Portable/Errors/MessageID.cs

```diff
@@ -127,7 +127,7 @@ internal enum MessageID
         IDS_FeatureTuples = MessageBase + 12711,
         IDS_FeatureOutVar = MessageBase + 12713,
 
-        // IDS_FeatureIOperation = MessageBase + 12714,
+        IDS_FeaturePragmaWarningEnableOrSafeOnly = MessageBase + 12714,
```

Comment:
>IDS_FeaturePragmaWarningEnableOrSafeOnly [](start = 8, length = 40)

What's the benefit of using a discrete feature. Feels like `IDS_FeatureNullableReferenceTypes` would be good for this, since tightly related to nullability.